sites/fast-website/src/app/data/feature.data.ts

```diff
@@ -1,65 +1,133 @@
+interface FeatureLink {
+    anchorText: string;
+    url: string;
+}
+
 export interface FeatureCardData {
-    body: string;
-    documentationLink: string;
-    githubLink: string;
     header: string;
+    body: string;
+    links?: FeatureLink[];
 }
 
 export const featureCardData: FeatureCardData[] = [
     {
+        header: "Composition",
         body:
             "Create new component compositions by nesting, styling base components, and passing unhandled props. The combinations are endless.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Composition",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Design Systems",
         body:
             "Use Fluent by default or customize design system properties to make it your own. Or, create your own design system to use with FAST components.",
-        documentationLink: "#",
-        githubLink: "#",
-        header: "Design Systems",
+        links: [
+            {
+                anchorText: "View GitHub",
+                url: "#",
+            },
+            {
+                anchorText: "Read Documentation",
+                url: "#",
+            },
+        ],
     },
     {
+        header: "Technology",
         body:
             "Out of the box, FAST components are built on React, but you can build components on any modern framework using the FAST system.",
```

Comment:
This text needs to change. Let me work something up - we should not reference React anywhere as these are for WC's.

Reply:
What about something like, "FAST web components are built on the core technologies of the web to work in any scenario.  Use them on their own or with your favorite framework."

Reply:
Sure, let’s start there, thx!

Reply:
Unresolving, I don't see this as having been changed.

Reply:
I would have sworn I updated that.  Thanks for catching that, Jane.  Fixed.